helmfile/values/falco-exporter.yaml.gotmpl

```diff
@@ -1,3 +1,12 @@
+image:
+  tag: 0.3.0
+  pullPolicy: IfNotPresent
+
+resources: {{- toYaml .Values.falco.falcoExporter.resources | nindent 2  }}
```

Comment:
falcoExporter uses it own chart. I would suggest to not keep it under falco values t.ex: .Values.falcoExporter.resources
Does it make sense?

Reply:
I thought so aswell untill i saw that falco-sidekick is under .Values.falco.falcoSideKick. So I went with this approch just to keep it consistent.

Reply:
ok